### ISSUE TITLE
name official successor app

### ISSUE THREAD

**Issue by core-code** [scrod/nv#384]
since the last release has been in 2011, maybe it is time to put a notice on the homepage blessing an official successor app, e.g. nvALT?

**Comment by chrowe** [comment]
Seems like nvALT has not been updated in a couple years either. I guess https://brettterpstra.com/2019/06/07/nvultra-is-nvultra/ is where new work is going.

**Comment by jamesnotjim** [comment]
I moved from NV to [nvALT](https://brettterpstra.com/projects/nvalt/) to [Quiver](https://happenapps.com/#quiver). I'm on he list to learn more about [nvUltra](https://nvultra.com/). Perhaps that is the official successor. I still have a lot of data in both NV and nvALT. But I've slowly been migrating it to Quiver.

**Comment by werediver** [comment]
I can recommend [Deft](https://github.com/jrblevin/deft) for Emacs. Works nicely with `org-mode` and `org-roam`. But it's Emacs, so... be warned 🙂

**Comment by paultroop** [comment]
It's a real shame that somebody isn't prepared to update this, given how many people find it so useful. I for one would be happy to donate to a programmer prepared to work on it. I'm sure many others are in the same boat.

**Comment by omundy** [comment]
As of at least MacOS Ventura I was unable to get Notational Velocity to run. nvUltra seems to be at a dead end so I tried ~~nvALT which was able to access the nv database and seems to work effectively.~~ **See note below...**

**Comment by scrod** [comment]
the app should run fine, you can just build it yourself in Xcode and run it. You’ll get a couple of bug fixes in the process. There’s nothing stopping you, really.

**Comment by omundy** [comment]
Scratch what I said above, nvAlt has issues... it was showing very old versions of notes. Do not use it. 

I was able to get Notational Velocity to work without problems (thanks to this [thread](https://twitter.com/chockenberry/status/1333566067113750529)) by using Safari to re-download the app, then allow it in System Settings.

@scrod thanks. I'm primarily C# and Javascript. Could you point me to reliable instructions for doing that?

**Comment by jamesnotjim** [comment]
I ended up migrating first to Quiver (https://yliansoft.com/) and then to
Joplin (https://joplinapp.org/). I recommend Joplin to any former nv and
nvALT users.

**Comment by core-code** [comment]
the only point of this bug report is a request for the original authors like @scrod to bless one of the actively maintained forks as the successor app. so that systems like homebrew and apps like macupdater can automatically update users to a newer app that actually works. 

if there is demand for a "lets chat about possible alternatives" thread, i suggest opening an additional issue here. thanks.

**Comment by jptechco** [comment]
I agree—it would be great to see @scrod affirm an actively maintained fork as the successor to NV, especially for users looking for something that continues to work on modern macOS.

I’m hopeful he’ll consider [Kinetic Notes](https://kineticnotes.org), a universal build and spiritual successor to his fantastic work on Notational Velocity - an app I used for years until I started getting warnings about Rosetta's EOL.  I am the contributor behind KN, and I’ve been working to carry on NV's simplicity and philosophy while modernizing the app.
